resources/ctia-default.properties

```diff
@@ -12,6 +12,8 @@ ctia.store.sql.db.subname=/tmp/ctia-h2-db;DATABASE_TO_UPPER=false
 ctia.store.sql.db.delimiters=
 ctia.store.es.uri=http://localhost:9200
 ctia.store.es.indexname=ctia
+ctia.producer.es.uri=http://localhost:9200
 ctia.producer.type=es
-ctia.producer.es.uri=http://127.0.0.1:9200
 ctia.producer.es.indexname=ctia_events
+ctia.producer.es.slicing.strategy=aliased-index
```

Comment:
I think the name of this property is confusing.  The user cares about if the CTIA instance is using time slicing, and not how its implemented.